laravel-app/app/Services/PostService.php

```diff
@@ -63,4 +64,30 @@ public function sharePost(UserPost $post): UserPost
 
         return $sharedPost;
     }
+
+    /**
+     * To update the image on post
+     */
+    public function updatePostImage(UserPost $post, UploadedFile $image): bool
```

Comment:
can you make this function descriptive as that it is return a boolean

Reply:
also i was thinking is this for updating image? then maybe void?

Reply:
oh, if its bool why cant you make it more descriptive as what it should be. the function name is updatePostImage

Reply:
Yes sir, I saw a stakoverflow thread about the naming convention on function that returns **bool**
https://stackoverflow.com/questions/1370840/naming-conventions-what-to-name-a-method-that-returns-a-boolean

I decided to rename the function `isPostImageUpdatable()`

Here is my call function on my `PostController.php`
 `$updatedImage = $this->postService->isPostImageUpdatable($post, $image);`

It is much more readable on what does the function returns.